src/pages/version/Metadata.tsx

```diff
@@ -121,7 +121,7 @@ export const Metadata: React.VFC<Props> = ({ loading, version }) => {
       )}
       {!isPatch && (
         <MetadataItem>
-          Github Commit:{" "}
+          Github commit:{" "}
           <InlineCode
             data-cy="version-github-commit"
             href={getGithubCommitUrl(owner, repo, revision)}
```

Comment:
Below, do you think the text should read "Commit queue position"? Or maybe "Commit Queue position"?

Reply:
Updated so it consistently uses sentence cases.